src/utils/options.ts

```diff
@@ -240,6 +245,7 @@ export {
   agentPort,
   connConnectTime,
   recoveryCodeFile,
+  recoveryCodeOutFile,
```

Comment:
What would be the problem with re-usung the original option `recoveryCodeFile`? Is there fundamental ambiguity here?

Reply:
We don't want to reuse the input recovery code file path, as that means it may try to overwrite the existing recovery code file. Additionally, the user may want to customise the output directory so it makes sense to keep these separate.

Reply:
Its better that the difference is explicit. While yes, we can't have an input and an output function at the same time, How would we tell the input apart from output? Check that we're imputing a path rather than a BIP39 code? Switching that way is way to implicit for my liking.

Reply:
I'll make sure that they are nicely documented. The `recovery-code-out-file` option by necessity only applies when needed. When not needed it is just ignored. For it's usage there are two cases

1. node generated a new recovery code for whatever reason - so rather than printing it out we divert the recovery code to a file specified by that path.
2. Node started with existing state - Do nothing, there was no recovery code to output. 

For case 2, I'd rather just have it silently do nothing. The service will have a static configuration. So first time it starts it needs to not output the code to the logs, but for all subsequent starts we don't need to handle it. But in both the initial and subsequent starts the `recovery-code-out-file` will be set.


This option needs to be separate from the `recovery-code-file` since their intentions *will* conflict depending on state. Starting a node without state will need a clear difference between inputting a recovery code, or outputting it to a file.

Reply:
I've updated the description to indicate that one is for reading the recovery code input, and the other is only used to divert the recovery code output.